plugins/Marketplace/config/test.php

```diff
@@ -83,11 +83,37 @@
 
         $service->authenticate($accessToken);
 
-        function removeReviewsUrl($content)
+        function updatePluginUrlsForTests(&$plugin)
+        {
+            if (!empty($plugin['shop']['reviews']['embedUrl'])) {
+                $plugin['shop']['reviews']['embedUrl'] = '';
+            }
+
+            if (!empty($plugin['coverImage'])) {
+                $plugin['coverImage'] = preg_replace(
+                    [
+                        '@^https?://.*?/([^/]*?)/images/([^/]*?)/(.*?)$@',
+                        '@^https?://.*?/img/categories/(.*?)$@i',
+                    ],
+                    [
+                        'plugins/Marketplace/tests/resources/images/plugins/$1/images/$2/$3',
+                        'plugins/Marketplace/tests/resources/images/categories/$1',
+                    ],
+                    $plugin['coverImage'],
+                    1
+                );
+            }
+        }
+
+        function updatePayload($content)
```

Comment:
Also update the name of this, so its intent is clear and you don't have to read the code to understand.

Reply:
Updated method names and added inline comments to better explain what they do.